src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -909,6 +909,32 @@ export class BaseEditCourseForm extends React.Component {
               className="course-skill"
             />
             )}
+            <Field
+              name="organization_short_code_override"
+              component={RenderInputTextField}
+              label={<FieldLabel text="Organization Short Code Override" optional />}
+              extraInput={{ onInvalid: this.openCollapsible }}
+              disabled={disabled}
+              optional
+            />
+            <Field
+              name="organization_logo_override_url"
+              component={ImageUpload}
+              label={(
```

Comment:
This field is also optional.

Reply:
Optional is already added in this field
https://github.com/openedx/frontend-app-publisher/pull/731/files#diff-d7038889932631eebe7e5c1f8a90c26c167f366f9deb7b22297382548710c24aR936

Are you talking about this or any other field?

Reply:
No, I meant both of the org override fields.

Reply:
Both are already optional
https://github.com/openedx/frontend-app-publisher/pull/731/files#diff-d7038889932631eebe7e5c1f8a90c26c167f366f9deb7b22297382548710c24aR936

https://github.com/openedx/frontend-app-publisher/pull/731/files#diff-d7038889932631eebe7e5c1f8a90c26c167f366f9deb7b22297382548710c24aR918